03week/airplane/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>UX Atlantic</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+
+  <body>
+    <header>
+
+      <!-- Make this div float the a tags -->
+      <div class="navbar">
+        <img src="./img/logo.png" class="logo" alt="logo">
+        <div class="navbarright">
+        <ul class="mainFeatures">
+        <li><a href=#><img src="./img/home-icon.png"></a>Home</li>
+        <li><a href=#><img src="./img/deal-icon.png"></a>Travel Deals</li>
+        <li><a href=#><img src="./img/schedule-icon.png"></a>Flight Schedule</li>
+        <li><a href=#><img src="./img/login-icon.png"></a>Register/Login</li>
+        </ul>
+        </div>
+      </div>
+    </header>
+
+    <main>
+      <article class="article1">
+          <div class="topbuttons">
+              <button><input type="image" class="buttonpadding" src="./img/plane-icon.png"/></button>
+              <button><input type="image" class="buttonpadding" src="./img/hotel-icon.png"/></button>
+              <button><input type="image" class="buttonpadding" src="./img/car-icon.png"/></button>
+            </div>
+
+<div class="everythingbuttopbuttons">
+          <div class="departarrive">
+              <input class="departbox" type="text" placeholder="Depart">
```

Comment:
things like this where you are only calling a class once, I would recommend using an ID instead